wallstation_modules/revertbeheading/code/head.dm

```diff
@@ -0,0 +1,4 @@
+// Editing: code/modules/surgery/bodyparts/head.dm
+/obj/item/bodypart/head
+    // allows dismembering heads
+	can_dismember = TRUE
```

Comment:
Inconsistent embeds are bad, if linters weren't down they'd probably error here

Reply:
no clue why it shows up like this on github, in editor it's embedded once on each line, i removed the comment as its just holdover from the massmeta revert i copied this from